internals/core_models.py

```diff
@@ -845,8 +845,6 @@ def _get_changes_as_amendments(self):
       if new_val != old_val:
         if (new_val == '' or new_val == False) and old_val is None:
           continue
-        new_val = convert_enum_int_to_string(prop_name, new_val)
-        old_val = convert_enum_int_to_string(prop_name, old_val)
         amendments.append(
             review_models.Amendment(field_name=prop_name,
             old_value=str(old_val), new_value=str(new_val)))
```

Comment:
Amendment field values are all converted to strings, so when we want to reference them for converting enums to readable form, we need to consider this.